### PR TITLE
add LongAtomTest

### PR DESCRIPTION
**Pull request by reymont**
Add LongAtomTest example to display the usage of volatile.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,67 @@ And then run the Jar inside the container
 docker run -i -v /tmp/jar:/data cortinico/java8-32bit java -jar myapp.jar
 ```
 
+## Run LongAtomTest example
```
Comment (cortinico) [inline README.md]: Can you change this to 

```
## How to run a sample Java File
```

File: README.md [hunk 2]
```diff
@@ -19,6 +19,67 @@ And then run the Jar inside the container
 docker run -i -v /tmp/jar:/data cortinico/java8-32bit java -jar myapp.jar
 ```
 
+## Run LongAtomTest example 
+
```
Comment (cortinico) [inline README.md]: Can you add a bit of description here, like:

```
Let's try to run a sample .java file inside the docker container. You can use a sample java file, LongAtomTest.java, pasted below.
```

**Review by cortinico — CHANGES_REQUESTED** [review]
Thanks for the PR <3
Here just a couple of comments ;)

**Comment by reymont** [comment]
the comments has changed.

**Pull request merged**